veascan-web/src/components/ColoredLabel.tsx

```diff
@@ -0,0 +1,37 @@
+import { Tag } from "@kleros/ui-components-library";
+import React from "react";
+import styled from "styled-components";
+
+interface ColoredLabelProps {
+  variant: string;
+  text: string;
+}
+
+const ColorTag = styled(Tag)<{ backgroundColor: string }>`
```

Comment:
Add style to avoid changing the cursor when hovering. (Right now is getting the pointer from the Tag)